site/guide/manage-users.qmd

```diff
@@ -11,16 +11,19 @@ quart
 
 2. Go to **Settings** in the sidebar and select **Profile**.
 
-3. You can find the following at your profile page.
+3. You can find the following on your profile page.
    - Profile details 
    - Roles attached to your account
+   - Groups assigned to your account
    - Values for API Key and Secret Key
 
 ## Prerequisites
 
 To perform these next steps, you must be a group administrator or a superuser.
 
-## Invite new users
+## User invitations
+
+### Invite new users
```

Comment:
This is missing a step after 2. You need to go to **Organization** first.

Reply:
@nrichers - I don't see the need to go to Org first for this one too - please help review again

https://www.loom.com/share/0ee070c1800841da911b4da8ad2131f3?sid=253baec2-3ef1-4c2b-ac96-9907b93df2f2

Reply:
Yes, you are correct — I wasn't aware that this option to invite new users was available directly from the **Settings** screen. Thank you for the helpful recording.